base_layer/core/src/proof_of_work/lwma_diff.rs

```diff
@@ -186,52 +169,51 @@ mod test {
 
     #[test]
     fn lwma_limit_difficulty_change() {
-        let mut dif = LinearWeightedMovingAverage::new(5, 60, 1);
+        let mut dif = LinearWeightedMovingAverage::new(5, 60, 1.into(), 60 * 6);
         let _ = dif.add(60.into(), 100.into());
-        let _ = dif.add(10_000_000.into(), 200.into());
+        let _ = dif.add(10_000_000.into(), 100.into());
         assert_eq!(dif.get_difficulty(), 17.into());
-        let _ = dif.add(20_000_000.into(), 216.into());
-        assert_eq!(dif.get_difficulty(), 10.into());
+        let _ = dif.add(20_000_000.into(), 16.into());
+        assert_eq!(dif.get_difficulty(), 12.into());
     }
 
     #[test]
     // Data for 5-period moving average
     // Timestamp: 60, 120, 180, 240, 300, 350, 380, 445, 515, 615, 975, 976, 977, 978, 979
     // Intervals: 60,  60,  60,  60,  60,  50,  30,  65,  70, 100, 360,   1,   1,   1,   1
     // Diff:     100, 100, 100, 100, 100, 105, 128, 123, 116,  94,  39,  46,  55,  75, 148
-    // Acum dif: 100, 200, 300, 400, 500, 605, 733, 856, 972,1066,1105,1151,1206,1281,1429
```

Comment:
I will keep this line here for future ref. Even thou its needed here. We will need this when going back to achieved difficulty.  Even thou achieved difficulty is not currently used for now. When implementing pruning we need to swop back to achieved difficulty.

Reply:
It can be calculated from the line above or retrieved from history, but I've put it back regardless